packages/adminrouter/docker/Dockerfile

```diff
@@ -8,9 +8,9 @@ ENV OPENRESTY_VERSION=1.13.6.1 \
     OPENRESTY_COMPILE_OPTS="" \
     VENV_DIR=/usr/local/venv \
     AR_BIN_DIR=/usr/local/adminrouter/ \
-    VTS_MODULE_DIR=/usr/local/src/jongiddy-nginx-module-vts-96b1562 \
-    VTS_MODULE_DOWNLOAD_SHASUM=37f2c0fe8d782daf4bb90cdf89c8c6b3a28462fc \
-    VTS_MODULE_DOWNLOAD_URL=https://github.com/jongiddy/nginx-module-vts/tarball/96b1562d0fb9387be120ccff3c5837890a5f8723 \
+    VTS_MODULE_DIR=/usr/local/src/jongiddy-nginx-module-vts-3686706 \
+    VTS_MODULE_DOWNLOAD_SHASUM=785a1a412355dd78c9b0c56318ac5e2fa7428292 \
+    VTS_MODULE_DOWNLOAD_URL=https://github.com/jongiddy/nginx-module-vts/tarball/36867068fd0a20f0ce1c0b01bb1539be70a7224f \
```

Comment:
Not-related but if you do make a change... I'd really like to see a comment here which links to a JIRA to take source code for DC/OS away from `/jongiddy` and into `/mesosphere` or `/dcos`.

Reply:
I didn't want to take it under the Mesosphere umbrella. The preferred route was for the fixes to be merged into the upstream dependency. So far, there's been no response to my PR's (or any PR's since mid-2018). So maybe we do need to maintain a Mesosphere-specific version.

Reply:
Thanks for the work @jongiddy :-)